services/web/server/src/simcore_service_webserver/storage/_rest.py

```diff
@@ -524,10 +534,13 @@ async def abort_async_job(request: web.Request) -> web.Response:
 @permission_required("storage.files.*")
 @handle_data_export_exceptions
 async def get_async_job_result(request: web.Request) -> web.Response:
+    class _PathParams(BaseModel):
+        job_id: UUID
```

Comment:
I would defien this at the top of the module and just use it, since it's private, sintead of defingng it when calling the handler

Reply:
@bisgaard-itis